zeppelin-web/src/app/jobmanager/jobmanager.filter.js

```diff
@@ -38,6 +39,17 @@
         });
       }
 
+      if (isSortByAsc === true) {
+        filterItems = _.sortBy(filterItems, function(sortItem) {
+          return sortItem.noteName;
+        });
+      } else {
+        filterItems = _.sortBy(filterItems, function(sortItem) {
```

Comment:
And If sorting by just an property, you can call it as more simple form like `_.sortBy(filterItems, ['noteName']); `